controller/api/utils.py

```diff
@@ -9,31 +9,31 @@
-        'fabled', 'famous', 'feline', 'finest', 'flaxen', 'folksy', 'frozen',
-        'gaslit', 'gentle', 'gifted', 'ginger', 'global', 'golden', 'grassy',
-        'hearty', 'hidden', 'hipper', 'honest', 'humble', 'hungry', 'hushed',
-        'iambic', 'iconic', 'indoor', 'inward', 'ironic', 'island', 'italic',
-        'jagged', 'jangly', 'jaunty', 'jiggly', 'jovial', 'joyful', 'junior',
-        'kabuki', 'karmic', 'keener', 'kindly', 'kingly', 'klutzy', 'knotty',
-        'lambda', 'leader', 'linear', 'lively', 'lonely', 'loving', 'luxury',
-        'madras', 'marble', 'mellow', 'metric', 'modest', 'molten', 'mystic',
-        'native', 'nearby', 'nested', 'newish', 'nickel', 'nimbus', 'nonfat',
-        'oblong', 'offset', 'oldest', 'onside', 'orange', 'outlaw', 'owlish',
-        'padded', 'peachy', 'pepper', 'player', 'preset', 'proper', 'pulsar',
-        'quacky', 'quaint', 'quartz', 'queens', 'quinoa', 'quirky',
-        'racing', 'rental', 'rising', 'rococo', 'rubber', 'rugged', 'rustic',
-        'sanest', 'scenic', 'shadow', 'skiing', 'stable', 'steely', 'syrupy',
-        'taller', 'tender', 'timely', 'trendy', 'triple', 'truthy', 'twenty',
-        'ultima', 'unbent', 'unisex', 'united', 'upbeat', 'uphill', 'usable',
-        'valued', 'vanity', 'velcro', 'velvet', 'verbal', 'violet', 'vulcan',
-        'webbed', 'wicker', 'wiggly', 'wilder', 'wonder', 'wooden', 'woodsy',
-        'yearly', 'yeasty', 'yeoman', 'yogurt', 'yonder', 'youthy', 'yuppie',
-        'zaftig', 'zanier', 'zephyr', 'zeroed', 'zigzag', 'zipped', 'zircon',
+        'ablest', 'absurd', 'actual', 'aerial', 'allied', 'artful', 'atomic', 'august',
+        'bamboo', 'benign', 'blonde', 'blurry', 'bolder', 'breezy', 'bubbly', 'burley',
+        'candid', 'calmer', 'casual', 'cheery', 'classy', 'clever', 'convex', 'cubist',
+        'dainty', 'dapper', 'decent', 'deluxe', 'docile', 'dogged', 'drafty', 'dreamy',
+        'earthy', 'easier', 'echoed', 'edible', 'elfish', 'excess', 'exotic', 'expert',
+        'fabled', 'famous', 'feline', 'finest', 'flaxen', 'folksy', 'frisky', 'frozen',
+        'gaslit', 'gentle', 'gifted', 'ginger', 'global', 'golden', 'grassy', 'guided',
+        'haptic', 'hearty', 'hidden', 'hipper', 'honest', 'humble', 'hungry', 'hushed',
+        'iambic', 'iciest', 'iconic', 'indoor', 'inward', 'ironic', 'island', 'italic',
+        'jagged', 'jangly', 'jaunty', 'jicama', 'jiggly', 'jovial', 'joyful', 'junior',
+        'kabuki', 'karmic', 'keener', 'kiddie', 'kindly', 'kingly', 'klutzy', 'knotty',
+        'lambda', 'latest', 'leader', 'linear', 'lively', 'lonely', 'loving', 'luxury',
+        'madcap', 'madras', 'marble', 'mellow', 'metric', 'modest', 'molten', 'mystic',
+        'native', 'nearby', 'nested', 'newish', 'nickel', 'nimbus', 'nonfat', 'normal',
+        'oblong', 'oddest', 'offset', 'oldest', 'onside', 'orange', 'outlaw', 'owlish',
+        'padded', 'pastry', 'peachy', 'pepper', 'player', 'preset', 'proper', 'pulsar',
```

Comment:
I just google for "8-letter words starting with a," for example. I ignore anything negative or edgy. If those ones are too clunky as adjective-type words I can look for others.